src/delta.rs

```diff
@@ -131,7 +131,7 @@ where
             painter.paint_buffered_minus_and_plus_lines();
             state = State::HunkHeader;
             painter.set_highlighter();
-            if should_handle(&state, config) {
+            if config.line_numbers || should_handle(&state, config) {
```

Comment:
Do you think it would make sense to incorporate the `if config.line_numbers` condition into `should_handle` so that all the logic is in one place?